docs/course_materials/exercises/check_helper.py

```diff
@@ -0,0 +1,35 @@
+import os.path
+from pathlib import Path
+
+home = str(Path.home())
+
+def get_student_commands():
+    '''
+    Checks stutent history file for key commands.
+    If the history is longer than 1000 commands, checks only 1000 last commands.
+    '''
+    my_commands = []
```

Comment:
I tried to run the script, I don't have this file, maybe we can use the history command instead? 
and I was thinking, maybe instead of getting the last 1000 command, maybe we can check commands from a range of dates? like for example for the past 10 days? what do you think?

Reply:
Made the adjustments, it should run properly now. Please let me now if there are problems.
(Had a problem at first to since I'm using zsh)